coda.io.apexdomain.json

```diff
@@ -0,0 +1,37 @@
+{  
+    "providerId":"coda.io",
+    "providerName":"Coda",
+    "serviceId":"apexdomain",
+    "serviceName":"Coda Site",
+    "version": 1,
+    "syncPubKeyDomain" : "coda.io",
+    "logoUrl":"https://cdn.coda.io/icons/png/color/coda-192.png",
+    "description":"Allows serving Coda documents from your own domain (apex only).",
+    "variableDescription":"`ipAddress[1, 2, 3]` refers to the IP addresses of Coda's custom domain server. `docId` refers to the coda document. `environment` refers to the environment for Coda's custom domain server. `docId` refers to the coda document used for custom domain. `verification` refers to the TXT verification of doc ownership",
```

Comment:
The description looks out-of-date and has some repetition. Seems like `docId` is just `verification` now.